resources/test/fixtures/U012.py

```diff
@@ -0,0 +1,23 @@
+"foo".encode("utf-8") # b"foo"
+"foo".encode("u8") # b"foo"
+"foo".encode() # b"foo"
+
+string = "hello there"
+string.encode("utf-8") # string.encode()
```

Comment:
pyupgrade does not rewrite this: [rationale](https://github.com/asottile/pyupgrade/issues/138#issuecomment-496044965), [source](https://github.com/asottile/pyupgrade/blob/v3.2.0/pyupgrade/_plugins/default_encoding.py), [tests](https://github.com/asottile/pyupgrade/blob/v3.2.0/tests/features/default_encoding_test.py).